projects/plugins/jetpack/sal/class.json-api-post-base.php

```diff
@@ -1,74 +1,208 @@
+				esc_html(
+					sprintf(
+						/* translators: %s is the method name that has been called */
+						__( 'Call to undefined method %s', 'jetpack' ),
+						$name
+					)
+				)
+			);
 		}
 	}
 
-	public function __isset ( $name ) {
+	/**
+	 * Checking to see if a given property is set.
+	 *
+	 * @param string $name Property to check if set.
+	 *
+	 * @return bool
+	 */
+	public function __isset( $name ) {
 		return isset( $this->post->{ $name } );
 	}
 
+	/**
+	 * Defining a base get_like_count() function to be extended in the Jetpack_Post class.
+	 *
+	 * This will define a default value for the like counts on a post, if this hasn't been defined yet.
+	 *
+	 * @see class.json-api-post-jetpack.php
+	 */
 	abstract public function get_like_count();
+
+	/**
+	 * Defining a base is_liked() function to be extended in the Jetpack_Post class.
+	 *
+	 * This will define a default value for whether or not the current user likes this post, if this hasn't been defined yet.
+	 *
+	 * @see class.json-api-post-jetpack.php
+	 */
 	abstract public function is_liked();
+
+	/**
+	 * Defining a base is_reblogged() function to be extended in the Jetpack_Post class.
+	 *
+	 * This will define a default value for whether or not the current user reblogged this post, if this hasn't been defined yet.
+	 *
+	 * @see class.json-api-post-jetpack.php
+	 */
 	abstract public function is_reblogged();
+
+	/**
+	 * Defining a base is_following() function to be extended in the Jetpack_Post class.
+	 *
+	 * This will define a default value for whether or not the current user is following this blog, if this hasn't been defined yet.
+	 *
+	 * @see class.json-api-post-jetpack.php
+	 */
 	abstract public function is_following();
+
+	/**
+	 * Defining a base get_global_id() function to be extended in the Jetpack_Post class.
+	 *
+	 * This will define the unique WordPress.com-wide representation of a post, if this hasn't been defined yet.
+	 *
+	 * @see class.json-api-post-jetpack.php
+	 */
 	abstract public function get_global_id();
+
+	/**
+	 * Defining a base get_geo() function to be extended in the Jetpack_Post class.
+	 *
+	 * This will define a default value for whether or not there is gelocation data for this post, if this hasn't been defined yet.
+	 *
+	 * @see class.json-api-post-jetpack.php
+	 */
 	abstract public function get_geo();
 
+	/**
+	 * Returns an int which helps define the menu order for the post.
+	 *
+	 * @return int
+	 */
 	public function get_menu_order() {
 		return (int) $this->post->menu_order;
 	}
 
+	/**
+	 * Returns a string which represents the post's GUID.
+	 *
+	 * @return string
+	 */
 	public function get_guid() {
 		return (string) $this->post->guid;
 	}
 
+	/**
+	 * Returns a string which represents the post type.
+	 *
+	 * @return string
+	 */
 	public function get_type() {
 		return (string) $this->post->post_type;
 	}
 
+	/**
+	 * Returns an object which holds the terms associated with that post object.
+	 *
+	 * @return object
```

Comment:
This could be better described like
```suggestion
	 * Return the post's terms.
	 *
	 * @return object Mapping taxonomy name -> term name -> formatted term.
```